docs/compatibility.md

```diff
@@ -83,8 +82,26 @@ The following apps are supported by govuk-docker to some extent.
    - ✅ support
    - ✅ support-api
    - ✅ transition
+   - ❌ transition-config
    - ✅ travel-advice-publisher
    - ⚠ whitehall
       * Who knows, really - several tests are failing, lots pass ;-)
       * Rake task to [create a test taxon](https://github.com/alphagov/whitehall/blob/master/lib/tasks/taxonomy.rake#L11) for publishing is not idempotent
       * Placeholder images don't work as missing proxy for [/government/assets](https://github.com/alphagov/whitehall/blob/master/app/presenters/publishing_api/news_article_presenter.rb#L133)
+
+## Generic Ruby libraries
+
+The following apps are supported as part of a "generic-ruby-library" project.
```

Comment:
Similar point about "supported". It also be useful to define what qualifies for being a generic ruby project.

Reply:
And also a brief line about what generic-ruby-library actually does / how it's useful. 👍

Reply:
Sure. I've clarified this to explain in more detail.